### DISCUSSION TITLE
I'm getting tired of Insomnia breaking more than it fixes

### DISCUSSION THREAD

**Discussion by metal-gabe** [Kong/insomnia#6438] [Feedback]
Okay, so this is a soapbox rant, but I'm fed up to the point where I feel the need to post this. 

Ever since 2023.5.x, Insomnia has become more of a nuisance than a tool I used to enjoy. 

As somebody pointed out in another discussion, it seems like generic UI updates are the priority instead of other existing issues. 

With every update it seems like more things break instead of what gets fixed. 

The first time I noticed a major break was when a colleague updated from 2023.4.0 to >2023.4.1 and he lost all of his requests & collections. 

At this point, what am I paying the subscription for? To have a broken, unstable tool? 

I'm seriously considering moving over to Postman (even though their UI is cluttered af). 

Hopefully, enough people get fed up with Insomnia and their voices bubble up to the proper product & business decision-makers over at Kong/Insomnia. 

This "tool" is getting ridiculous.

**Comment by goretaivor** [comment]
I am getting tired of more and more cluttered UI which is the reason I switched from Postman.
Also the introduction of useless vertical space below home icon and right margin in left colon.
And Base Environment and Add Cookies options that I now have to view all the time for no reason.
And Git Sync at the bottom.
Why can't you have all these options somewhere hidden instead of sprinkled all over the place.
After I set these thing I don't want to look at them.
And I am not using them, so why do I have to look at them all the time.
It is distracting from the actual Request and Reponses data I want to work with,

**Comment by subnetmarco** [comment]
We are making continuous improvements to Insomnia at a much higher pace these days, and fixing more issues than ever before. I hope you will give it another try if you decided to stop using since you first opened this issue.
